src/main/js/controls/Bar/helpers/legendHelpers.js

```diff
@@ -49,15 +48,16 @@ const clickHandler = (graphContext, control, config, canvasSVG) => (
         }
     };
     legendClickHandler(element);
+    const isLegendSelected = isSelected(d3.select(element));
     updateShownTarget(config.shownTargets, item);
     canvasSVG
         .selectAll(`g[aria-describedby="${item.key}"]`)
-        .attr("aria-hidden", true);
+        .attr("aria-hidden", isLegendSelected);
```

Comment:
If you are not reusing the variable then you might want to directly call the function here?

Reply:
Updated here https://github.com/cerner/carbon-graphs/pull/140/commits/a5884626cdc15cd1f6c9143a66a8af7d004296f0.